src/Altinn.Platform/Altinn.Platform.Storage/Storage/appsettings.json

```diff
@@ -22,7 +22,10 @@
     "AccountName": "devstoreaccount1",
     "AccountKey": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",
     "StorageContainer": "servicedata",
-    "BlobEndPoint": "http://127.0.0.1:10000/devstoreaccount1"
+    "BlobEndPoint": "http://127.0.0.1:10000/devstoreaccount1",
```

Comment:
remember to update values in ops project with these vcalues for each environment if they differ based on env.